ewah/ewah_utils/yml_loader.py

```diff
@@ -16,11 +19,16 @@
 """
 
 
+
 class Loader(base_loader):
     def __init__(self, stream):
+        with create_session() as session
+            variable_keys = [var.key for var in session.query(Variable)]
+            variable_values = [var.val for var in session.query(Variable)]
+        variables = dict(zip(variable_keys, variable_values))
         self._root = os.path.split(stream.name)[0]
         # Enable Jinja2 in the yaml files
-        yaml_stream = StringIO(Template(stream.read()).render(env=os.environ))
+        yaml_stream = StringIO(Template(stream.read()).render(env=os.environ, variables=variables))
```

Comment:
Request (blocking): Can you rename it `airflow_variables` instead of `variables`? Makes it a bit more clear :)